public/js/custom-hj.js

```diff
@@ -6,6 +6,12 @@ jQuery(function($) {
     $('.gp-book-online').on('click', function () {
       hj('tagRecording', ['Book Online']);
     });
+    $('.repeat-prescription-online').on('click', function () {
```

Comment:
A touch client must support `click` events as the basic interaction and under the covers a `click` event is also being triggered. From my testing I've not found any problems. Is there is a specific test case I can run?